.github/workflows/alert_unsigned_commits.yml

```diff
@@ -0,0 +1,44 @@
+name: Alert Unsigned Commits
+on:
+  pull_request_review:
+    types: [submitted]
+  pull_request:
+    types: [reopened, synchronize]
+
+permissions:
+  pull-requests: write
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  alert-unsigned:
+    if: github.event.review.state == 'approved'
+    name: Alert Unsigned Commits
+    runs-on: ubuntu-latest
+    steps:
+      - name: Alert Unsigned Commits
+        run: |
+          env | grep -i GIT | sort  # TODO remove before merge
+          set -x
+          HEAD_SHA=${{ github.event.pull_request.head.sha }}
+
+          BRAVE_CORE_PATH=brave/brave-core
+          post_once() {
+              comments=$(gh -R "$BRAVE_CORE_PATH" pr view "$GITHUB_HEAD_REF" --json comments --jq ".comments[].body")
```

Comment:
This could be a lot of text. Could hit some limits, either on the API side, or the gh side, or the bash side. Could we narrow it down to comments by this user?

Reply:
Also the variable `comments` is only used once, so we could just pipe `gh` to `grep` and skip the variable. Up to you, whatever reads better.

Reply:
True. As per https://github.com/cli/cli/issues/5415 the cli results are currently limited to 100 items (comments). Assuming each comment is 500bytes, that's roughly 50KB which is not too bad. `gh pr view` does not have the ability to filter per user. Also note that those comments, are different from "PR review comments" (https://docs.github.com/en/rest/pulls/comments) which I think a PR might get too many from. Also this workflow is limited in that it runs only when an approving review is received, so it doesn't run that often. All in all, I think it is OK for what it does.